### DISCUSSION TITLE
Tablet not working

### DISCUSSION THREAD

**Discussion by Pixelend10** [ppy/osu#19352] [Q&A]
### Type

Game behaviour

### Bug description

The area of the tablet is the one selected by the drivers or windows, I used to play with a 20:11 area but is not working anymore both with tablet drivers on and off

### Version

2022.719.0-lazer

### Logs

[network.log](https://github.com/ppy/osu/files/9176070/network.log)
[performance.log](https://github.com/ppy/osu/files/9176071/performance.log)
[runtime.log](https://github.com/ppy/osu/files/9176072/runtime.log)
[updater.log](https://github.com/ppy/osu/files/9176073/updater.log)
[database.log](https://github.com/ppy/osu/files/9176074/database.log)
[legacy-ipc.log](https://github.com/ppy/osu/files/9176075/legacy-ipc.log)

**Comment by frenzibyte** [comment]
Do you see the cursor moving but fading away when dragging it? If so, please check out:
 - https://github.com/ppy/osu/discussions/19261
 - https://github.com/ppy/osu/discussions/19303
 - https://github.com/ppy/osu/issues/19245

Otherwise check if you have "Touch" disabled, and try enabling it to see if the cursor moves.

**Comment by Chariakin** [comment]
Hi, since latest update Osu:Lazer Have pretty same problem for me.

CTH-470 ( Tablet ) 

Using Internal Osu Tablet driver : Nope.

Problem : When i move my cursor, they freeze every 1-2 seconds ( calculated btw, i can make a video if u want )

Tried uninstall / reinstall Driver & OsuLazer.

**Reply by frenzibyte** [reply]
1. Reinstalling osu!(lazer) will never fix an issue.
2. Please try looking at what I posted above.

**Comment by cdwcgt** [comment]
I have no idea that if you use the lazer built-in settings to adjust the tablet
![image](https://user-images.githubusercontent.com/34775378/181018932-ce249735-154f-41a9-8d8f-a5fb31fe8953.png)
or not something else
I used to have this problem too, because lazer is using OpenTabletDriver
so your computer must has been setup to using OpenTabletDriver
https://opentabletdriver.net/Wiki
For windows, you just remove all driver on your tablet and install WinUSB for your tablet and lazer will handle

https://opentabletdriver.net/

**Comment by Chariakin** [comment]
Update : 

Video : https://www.youtube.com/watch?v=qOoiypfkLns 

problem is the same than old days the cursor freeze every 2 seconds. 

Not using internal osu tablet my area is set with Wacom driver. 
![image](https://user-images.githubusercontent.com/109980990/182029216-6b4e18f5-b343-40e3-8c8a-87bb5c6acc22.png)

Windows version : W11 21H2 ( build 22000.778 ) 

Wacom driver ( CTH-470/S )  5.3.5-3 
Osu Lazer : Latest version.

Computer : https://www.cdiscount.com/informatique/m/pc-portable-gamer-lenovo-legion-5-15ach6h-15-6/f-107530504-82ju00elfr.html?idOffre=1147778571#faqs 

i stay here for further answers

**Reply by frenzibyte** [reply]
Have you tried toggling off "High precision mouse" in settings.
